src/Compilers/Server/PortableServer/CoreClrCompilerServerHost.cs

```diff
@@ -15,17 +15,15 @@ namespace Microsoft.CodeAnalysis.CompilerServer
 {
     internal sealed class CoreClrCompilerServerHost : CompilerServerHost
     {
-        private readonly Func<string, MetadataReferenceProperties, PortableExecutableReference> _assemblyReferenceProvider = (path, properties) => new CachingMetadataReference(path, properties);
-        private readonly IAnalyzerAssemblyLoader _analyzerAssemblyLoader = CoreClrAnalyzerAssemblyLoader.CreateAndSetDefault();
+        public override IAnalyzerAssemblyLoader AnalyzerAssemblyLoader { get; }
 
-        public override IAnalyzerAssemblyLoader AnalyzerAssemblyLoader => _analyzerAssemblyLoader;
-
-        public override Func<string, MetadataReferenceProperties, PortableExecutableReference> AssemblyReferenceProvider => _assemblyReferenceProvider;
+        public override Func<string, MetadataReferenceProperties, PortableExecutableReference> AssemblyReferenceProvider { get; }
 
         internal CoreClrCompilerServerHost(string clientDirectory)
             :base(clientDirectory : clientDirectory, sdkDirectory: null)
         {
-
+            AssemblyReferenceProvider = (path, properties) => new CachingMetadataReference(path, properties);
+            AnalyzerAssemblyLoader = CoreClrAnalyzerAssemblyLoader.CreateAndSetDefault();
```

Comment:
Unfortunately i've been unable to remove this.  I tried rebasing and manually editing this change out but I can't get it correct.  I'll be more dilligent about this in the future :frowning: